src/Form/Select.php

```diff
@@ -4,14 +4,32 @@
 
 namespace MyParcelCom\JsonSchema\FormBuilder\Form;
 
-use MyParcelCom\JsonSchema\FormBuilder\Properties\Meta\MetaFieldType;
+use MyParcelCom\JsonSchema\FormBuilder\SchemaProperties\Meta\MetaFieldType;
+use MyParcelCom\JsonSchema\FormBuilder\Translations\LabelTranslationCollection;
 use Override;
 
 class Select extends ChoiceField
 {
+    public function __construct(
+        string $name,
+        string $label,
+        OptionCollection $options,
+        bool $isRequired = false,
+        ?string $help = null,
+        ?LabelTranslationCollection $labelTranslations = null,
+        protected ?string $initialValue = null,
+    ) {
+        parent::__construct($name, $label, $options, $isRequired, $help, $labelTranslations);
```

Comment:
Can you identify the properties (like `name: $name`)? It is very handy when reading the code and we make sure that name and label, being the same type, are not swapped by mistake. phpStorm can help you easily insert the identifiers.